static/app/components/feedback/useFetchFeedbackData.tsx

```diff
@@ -1,6 +1,6 @@
 import {useEffect, useMemo} from 'react';
 
-import hydrateEventTags from 'sentry/components/feedback/hydrateEventTags';
+import hydrateFeedbackTags from 'sentry/components/feedback/hydrateFeedbackTags';
```

Comment:
Seems like an unrelated change? was this intentional?

Reply:
renamed the file! bc it's not only event tags